components/Navbar.vue

```diff
@@ -1,27 +1,26 @@
+<script setup lang="ts">
+const navItems = [
+  { name: "Domains", to: "/" },
+  { name: "Second", to: "/" },
+  { name: "Third", to: "/" },
+]
+</script>
+
 <template>
   <div class="flex h-16 w-screen flex-row items-center justify-between gap-8 bg-purpur-700 px-8 text-white">
     <div class="flex flex-row items-center gap-8">
       <NuxtLink to="/">
         <h1 class="title text-3xl font-bold text-white transition-colors hover:text-white/80">No Brain</h1>
       </NuxtLink>
+
       <div class="flex w-80 flex-row gap-2">
-        <template v-for="item in menuItems" :key="item">
-          <nuxt-link :to="item.to">
-            <h2 class="hover:bg-whie/20 rounded-lg p-3 font-semibold transition-colors hover:bg-white/20">
-              {{ item.name }}
-            </h2>
-          </nuxt-link>
-        </template>
+        <nuxt-link v-for="item in navItems" :key="item" :to="item.to">
```

Comment:
```suggestion
        <nuxt-link v-for="item in navItems" :key="item.name" :to="item.to">
```
![image](https://github.com/itswilliboy/have-no-bra.in/assets/83978878/77435efe-160f-4f68-ac24-58573dd82b4b)
Since `:key` should be a string or a number.

Reply:
Ah yeah sure, I just copy pasted what was on template and didn't really see what it was. Small oversight.